app/views/albums/_form.html.erb

```diff
@@ -21,7 +21,7 @@
        <% end %>
     </div>
     <br>
-    <strong><label>Photos:</label></strong><br>
+    <strong><label>Album Photos:   </label></strong><br>
```

Comment:
Naresh what is this, why are u giving spaces unnecessary , don't you love money wanrt to  loose jobl,lm,;,,;,;